docs/stdlib/bytes.rst

```diff
@@ -124,6 +127,24 @@ Bytes
 
 ---------
 
+.. eql:function:: std::to_bytes(s: str, encoding: optional str="utf-8") \
```

Comment:
Should 'encoding' be an enum?

Reply:
That sounds like it would be annoying for us with little gain

Reply:
We use enum for alphabet in the parallel PR for base64. Enum enables strict typing and removes ambiguity about which case the string literal should be in.

Reply:
I agree with Sully that this would be annoying to maintain as an enum.  The list is quite large, could be Postgres-version dependent, and is actually [dynamic](https://www.postgresql.org/docs/current/sql-createconversion.html).

> removes ambiguity about which case the string literal should be in

Its case-insensitive (should probably document this fact).  Also, common variations of encoding names are accepted, e.g 'utf-8' and 'utf8'.

Reply:
I don't like this. Exposing a standard library function that's tied to the underlying postgres version, poorly documented because of that, can fail at runtime when you deploy to prod, etc. I'd much prefer exposing a narrow set of encodings that we can support across a wide list of versions guaranteed. I'm -1 on adding this in its current form.

Reply:
Are you suggesting we only support utf-8?  Because I'm not sure which criteria to apply when considering which encodings are "blessed" then.

Reply:
The ones supported by all versions of Postgres we support? I assume it's unlikely for Postgres to drop an encoding they already support in some future version?

Reply:
We can gradually add more encodings in major versions as we shift the window of supported postgreses. But it would be weird if some encoding works in prod and doesn't in local, or vice versa. We have to somehow control the list of encodings we support and document.

Reply:
I decided to just omit the whole encoding thing for now.